src/index.html

```diff
@@ -4,9 +4,282 @@
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">cмарфоны и телефоны</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__link" href="#">ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog main__catalog">
+        <article class="card catalog__card" data-qa="card">
+          <div class="card__image">
+            <img
+              class="card__photo"
+              src="images/imac.jpeg"
+              alt="photo"
+            >
+          </div>
+          <div class="card__content">
+            <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+            <p class="card__subtitle">Код товара: 195434</p>
+            <div class="card__rating">
+              <div class="card__stars stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <h3 class="card__reviews">Отзывов: 5</h3>
+            </div>
+            <div class="card__valuation">Цена: <span class="card__price">69 999 грн</span></div>
+            <div class="button-buy card__button" data-qa="card-hover">
+              <a href="#" class="button-buy__link">Купить</a>
+            </div>
+          </div>
+
+        </article>
+
+        <article class="card catalog__card" data-qa="card">
+          <div class="card__image">
+            <img
+              class="card__photo"
+              src="images/imac.jpeg"
+              alt="photo"
+            >
+          </div>
+          <div class="card__content">
+            <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+            <p class="card__subtitle">Код товара: 195434</p>
+            <div class="card__rating">
+              <div class="card__stars stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <h3 class="card__reviews">Отзывов: 5</h3>
+            </div>
+            <div class="card__valuation">Цена: <span class="card__price">69 999 грн</span></div>
```

Comment:
таким чином потрібно пофіксити всі каточки
```suggestion
            <div class="card__valuation">
              Цена: <span class="card__price">69 999 грн</span>
            </div>
```